src/XPurchase/VirtoCommerce.XPurchase/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -14,6 +18,7 @@ public static class ServiceCollectionExtensions
     {
         public static IServiceCollection AddXPurchase(this IServiceCollection services, IGraphQLBuilder graphQlbuilder)
         {
+            services.AddSingleton<Func<SignInManager<ApplicationUser>>>(provider => () => provider.GetService<SignInManager<ApplicationUser>>());
```

Comment:
Please remove, as now we registered this in the platform. I've already updated dependency

Reply:
Removed